File: Programming EF Code First/chapter5/Chapter5 CSharp/BAGA_FluentAPI/DataAccess/PersonPhotoConfiguration.cs
using System.Data.Entity.ModelConfiguration;
using Model;
using System.ComponentModel.DataAnnotations;

namespace DataAccess
{
  public class PersonPhotoConfiguration : EntityTypeConfiguration<PersonPhoto>
  {
    public PersonPhotoConfiguration()
    {
      HasKey(p => p.PersonId);

      HasRequired(p => p.PhotoOf)
        .WithRequiredDependent(p => p.Photo);

      Property(p => p.Photo)
        .HasColumnType("image");

      ToTable("People");
    }
  }
}

File: Programming EF Code First/chapter6/Chapter6/Chapter 6 VB/Fluent/DataAccess/PersonPhotoConfiguration.vb
Imports System.Data.Entity.ModelConfiguration
Imports Model
Imports System.ComponentModel.DataAnnotations

Namespace DataAccess
  Public Class PersonPhotoConfiguration
	  Inherits EntityTypeConfiguration(Of PersonPhoto)
	Public Sub New()
	  HasKey(Function(p) p.PersonId)

	  HasRequired(Function(p) p.PhotoOf).WithRequiredDependent(Function(p) p.Photo)

      Me.Property(Function(p) p.Photo).HasColumnType("image")

      Me.ToTable("People")
	End Sub
  End Class
End Namespace